.aws/src/event-rules/prospect-events/prospectEventRules.ts

```diff
@@ -39,17 +41,33 @@ export class ProspectEvents extends Resource {
     super(scope, name);
 
     // pre-existing queues (prod and dev) created by prospect-api
-    this.sqs = new sqs.DataAwsSqsQueue(this, `prospect-${config.environment}-queue`, {
-      name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue`
-    })
+    this.sqs = new sqs.DataAwsSqsQueue(
+      this,
+      `prospect-${config.environment}-queue`,
+      {
+        name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue`,
+      }
+    );
+
+    this.sqsDlq = new sqs.DataAwsSqsQueue(
+      this,
+      `prospect-${config.environment}-dlq`,
+      {
+        name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue-Deadletter`,
+      }
+    );
 
-    this.sqsDlq = new sqs.DataAwsSqsQueue(this, `prospect-${config.environment}-dlq`, {
-      name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue-Deadletter`
+    this.snsTopic = new sns.SnsTopic(this, 'prospect-event-topic', {
+      name: `${config.prefix}-ProspectEventTopic`,
     });
 
     this.createProspectEventRules();
```

Comment:
similar question here - should this function be refactored to something like `createProspectGenerationEventRules`? should we move the above sqs code into a separate function?

Reply:
We could do a slightly bigger refactor than that and pull out the SQS creation (for prospect-generation events) and the make the SNS creation a general function. That way the SNS creation is not tied to our dimiss-prospect event.

Reply:
i like the direction proposed. i'll leave it up to you to decide on where the line is between a refactor now and future improvements.